### PR TITLE
Update to 4.2.2 to pick up node-express-server generator

### LINKED ISSUE #82: Update to 4.2.2 to pick up node-express-server generator, for Eclipse
Update to 4.2.2 to pick up node-express-server generator, for Eclipse.

We need the latest Node-server generator.  The original is deprecated.
There are some other interesting fixes that here that we can take advantage of like jaxrs-spec server codegen specifically targetting Open Liberty projects.

### PR DESCRIPTION
**Pull request by keithchong**
Fixes #82 

Signed-off-by: Keith Chong kchong@ca.ibm.com

### CODE REVIEW

**Comment by sghung** [comment]
I confirmed with Keith that the .classpath entry isn't necessary as it isn't required to be a plugin runtime jar. The removal in the PR is for cleanup purposes.

File: dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Constants.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ public Constants() {
             "jaxrs-resteasy",
             "jaxrs-resteasy-eap",
             "spring"},
-        {"Node.js", "nodejs-server"},
+        {"Node.js", "nodejs-express-server", "nodejs-server-deprecated"},
```
Comment (sghung) [inline dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Constants.java]: Why do we need to add the nodejs-server-deprecated? If it is going (or is) deprecated, shouldn't we stop it from showing?

**Comment by sghung** [comment]
@keithchong I am using Eclipse 2019-12 (4.14.0) and I see compilation errors after creating a node js express server stub. A lot of errors complaining about missing semi colons. Do you see the same issue?

Also, what context root did you use to verify the results? http://127.0.0.1:32805/pets doesn't work. Also, in the automated tests, besides the verification of the files, can we call Codewind to verify the API is there (also to verify what context root should be used).

File: dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Constants.java (hunk 1 above)
Comment (keithchong) [reply dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Constants.java]: If the generator keeps it, we'll keep it.  We try to mirror what the command line generator has.  Another reason is that if someone wants to use it, they don't have to get a back-level version of the generator.

**Comment by keithchong** [comment]
@sghung , the codewind build itself is ok.  Those are validation errors coming from the WTP Javascript validator.  Try disabling the validator on the project or the Eclipse workspace.  You will not see these errors in VS Code.  

What you're doing is trying to generate the server into the current Codewind project.  The usability improvements will come next.  This may involve not allowing the project root to be the output folder.

I want the 4.2.2 update also to take advantage of the Liberty-specific code generation for Java JAX-RS.

**Pull request merged**